packages/vue/src/components/confirm-sign-up.vue

```diff
@@ -10,10 +10,14 @@ import { useAuthenticator } from '../composables/useAuth';
 import BaseFormFields from './primitives/base-form-fields.vue';
 
 const attrs = useAttrs();
+
+/** @deprecated Authenticator component events are deprecated and not maintained. */
 const emit = defineEmits(['confirmSignUpSubmit', 'lostCodeClicked']);
 
-const { isPending, error, codeDeliveryDetails } = toRefs(useAuthenticator());
-const { submitForm, updateForm, resendCode } = useAuthenticator();
+// `useAuthenticator` is casted for temporary type safety on this file.
+const props = useAuthenticator();
```

Comment:
Missing cast?

Reply:
Thanks for the catch, updated in 414dc0370243cf43fa25195041d5bfa9d73d15a1 👍